packages/visualizations/src/components/utils/Card.svelte

```diff
@@ -47,6 +47,8 @@
         --spacing-75: 9px;
         --spacing-100: 13px;
         --border-color: #cbd2db;
+        --header-background-color: #ffffff;
+        --vertical-scroll-height: 20em;
         /* FIXME: Only using flex style to center source link */
         display: flex;
```

Comment:
Could a use a `flex-wrap: nowrap`. It's a direct consequence of this PR, but it becomes more obvious now. Else we can end up in horizontal layout in some cases.

Reply:
done